src/browser/pages/login.html

```diff
@@ -10,7 +10,7 @@
       http-equiv="Content-Security-Policy"
       content="style-src 'self'; script-src 'self' 'unsafe-inline'; manifest-src 'self'; img-src 'self' data:; font-src 'self' data:;"
     />
-    <title>{{APP_NAME}} login</title>
+    <title>{{APP_NAME}} {{I18N_LOGIN_TITLE}}</title>
```

Comment:
Re: keeping space...we do it here, so I think we should keep this pattern.

Reply:
Also should the app name be part of the login title?  Not sure if in some languages the app name might come after the word for `login`.

Reply:
ok. I will put the app name into the whole title.